crates/matrix-sdk-crypto/src/dehydrated_devices.rs

```diff
@@ -477,6 +477,10 @@ mod tests {
             !request.fallback_keys.is_empty(),
             "The dehydrated device creation request should contain some fallback keys"
         );
+
+        let device_keys: crate::types::DeviceKeys =
```

Comment:
Doesn't `request.device_keys.deserialize_as()` work here? Also please import `DeviceKeys` if you need them.

Reply:
changed to use `deserialize_as` (I didn't know about the function).  `mod tests` already imports `ruma::encryption::DeviceKeys`, so I can't import `crate::types::DeviceKeys` at the module-level.  Do I just `use crate::types::DeviceKeys` at the top of this function?  It looks like that's possible, but I've never seen that done, so I don't know if that's a "normal" thing to do.

Reply:
It's possible, you can also rename things if you import things using `use Foo as Bar`.